etc/adminhtml/system.xml

```diff
@@ -135,11 +134,31 @@
                     </field>
                     <field id="pricing_strategy" showInDefault="1" showInStore="1" showInWebsite="1" sortOrder="100" translate="label" type="select">
                         <label>DHL Pricing Strategy</label>
-                        <source_model>Reach\Payment\Model\Config\Source\Pricing</source_model>
+                        <source_model>Reach\Payment\Model\Config\Source\PricingStrategy</source_model>
+                        <depends>
+                            <field id="active">1</field>
+                        </depends>
+                    </field>
+                    <field id="clearance_mode" showInDefault="1" showInStore="1" showInWebsite="1" sortOrder="100" translate="label" type="select">
+                        <label>Clearance Mode</label>
+                        <source_model>Reach\Payment\Model\Config\Source\ClearanceMode</source_model>
+                        <depends>
+                            <field id="active">1</field>
+                        </depends>
+                    </field>
+                    <field id="end_use" showInDefault="1" showInStore="1" showInWebsite="1" sortOrder="100" translate="label" type="select">
+                        <label>End Use</label>
+                        <source_model>Reach\Payment\Model\Config\Source\EndUse</source_model>
```

Comment:
Please kindly make sure appropriate default values  (as per JIRA MAG-16) are set for all these fields. Thanks!

Reply:
Default values set. Can be confirmed in default.xml